checks/containers/check-singularity-mpi-module.py

```diff
@@ -0,0 +1,37 @@
+import os
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.simple_test
+class singularity_cray_mpich_bindings_test(rfm.RunOnlyRegressionTest):
+    descr = 'Test the singularity-bindings module with glibc'
+    valid_systems = ['lumi:gpu', 'lumi:standard']
+    valid_prog_environs = ['builtin']
+    num_tasks = 2
+    num_tasks_per_node = 1
+    modules = ['singularity-bindings']
+    reference = {
+        'lumi:gpu': {'bandwidth': (22180.76, -0.05, None, 'MB/s')}
+    }
+
+    @run_before('run')
+    def prepare_build(self):
+        self.sourcesdir = os.path.join(self.current_system.resourcesdir,
+                                       'singularity-binding-test')
+
+    @run_before('run')
+    def set_container_variables(self):
+        self.container_platform = 'Singularity'
+        self.container_platform.image = 'osu-debian-jessie.sif'
```

Comment:
Where this image comes from? Should it have osu bencharks inside?

Reply:
It comes from [here](https://sarus.readthedocs.io/en/stable/user/user_guide.html?highlight=mpich#native-mpi-support-mpich-based). I will add a comment.